ntoskrnl/ke/amd64/trap.S

```diff
@@ -611,7 +611,7 @@ PUBLIC KiRetireDpcListInDpcStack
 
     /* Switch stack and call the function */
     mov rsp, rdx
-    sub rsp, 40
+    sub rsp, 48
```

Comment:
Why do we need to reserve this much stack anyway ? How did it work before ?

Reply:
The caller needs to reserve 32 bytes for the callee (home space). Since on function entry the stack alignment is off by 8 bytes, a non-leaf function must allocate at least 40 bytes to make the stack aligned in the function body. This is what I replicated here first, but the new stack is aligned on function entry. 32 is probably fine, but those extra 16 bytes won't hurt anyone.

Reply:
Well, maybe add a comment there. Someone might see this code and wonder why we need 16 bytes more...